src/index.js

```diff
@@ -0,0 +1,177 @@
+
+/**********************1*********************/
+
+
+
+
+const data = [
+  { value: "1", label: "One" },
+  { value: "2", label: "Two" },
+  { value: "3", label: "Three" },
+];
+
+const values = data.reduce((values, { value }) => {
+  values.push(value);
+  return values;
+}, []);
+
+
+
+/**********************2*********************/
+
+
+async function getIndexes() {
+  return await fetch('https://api.coingecko.com/api/v3/indexes').then(res => res.json());
+}
+
+async function analyzeIndexes() {
```

Comment:
unecessary use of await keyword here, fetch promise is already resolved

 it's unnecessary to catch and throw the caught error immediately without any other added treatment

-  try {return getIndexes()} catch(e) {throw new Error('Unable to fetch indexes')}